src/components/MainHeaderComponent/MainHeaderComponent.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import {Icon, Menu} from 'antd'
+import './MainHeaderComponent.scss'
+
+
+const MainHeader = () => {
+  return (
+    <div className="main-header cc-container">
+      <div className="main-header__logo">
+        CConverter
+      </div>
+      <Menu
+        className="main-header__menu"
+        theme={'cc-purple'}
+        mode={'horizontal'}>
+        <Menu.Item key="1">
+          <a href="https://github.com/mad-rat/cconverter" title="Github Repo"  target="_blank" rel="noopener noreferrer">
+            <Icon type="github"/> Github
```

Comment:
Can we change `Github` to `GitHub` ? Here and in `a title`

Reply:
Yes, you can change it because it's out of scope. Elo Essa.

Reply:
xD

Reply:
Just kidding mate! xD 
 Fixed - thanks! 🦄